source/docs/magento2/add-new-attribute.md

```diff
@@ -0,0 +1,187 @@
+
+```graphql
+{
+  product(sku: "yourSKUhere") {
+    name
+    rate
+  }
+}
+```
+
+From this GraphQL query, you should get a JSON content that looks like this:
+
+```json
+{
+  "data": {
+    "product": {
+      "name": "Your product name",
+      "rate": // the value stored in your backoffice (Magento 2 in this case)
+    }
+  }
+}
+```
+
+### Overriding the files
+
+In **this** example, the files to be overridden are `ProductSynthesisFragment.gql` and `Synthesis.js` both located at `node_modules/front-commerce/src/web/theme/modules/ProductView/Synthesis/`. Copy and paste them into `mymodule/web/theme/modules/ProductView/Synthesis`.
+
+#### How to find which files to edit
+
+On your web page, right click on the element you wish to modify, then click on **Inspect Element**. In the inspector, you can look for the `className` that holds the elements you want to edit. Then you can search for that keyword in your `node_modules/front-commerce/src` folder and you will find the right files.
+
+We've included images to illustrate our example.
+![Inspector screenshot here](./assets/inspector-screenshot.png)
+
+![Synthesis screenshot here](./assets/synthesis-screenshot.png)
+
+### Updating the query
+
+In `ProductSynthesisFragment.gql`, add the following query line. It means that the application will request the `rate` field as well when sending the query. (Depending on the version of Front-Commerce you are using, the content might differ slightly.)
+
+```diff
+#import "theme/components/organisms/Configurator/ProductConfiguratorFragment.gql"
+#import "theme/modules/AddToCart/ProductStockFragment.gql"
+#import "theme/components/atoms/Typography/Price/ProductPriceFragment.gql"
+#import "theme/modules/ProductView/Details/ProductDetailsFragment.gql"
+
+fragment ProductSynthesisFragment on Product {
+  sku
+  name
+  ...ProductPriceFragment
+  description
++ rate
+  ...ProductConfiguratorFragment
+  ...ProductStockFragment
+  ...ProductDetailsFragment
+  configurations {
+    product {
+      sku
+      ...ProductPriceFragment
+    }
+  }
+}
+```
+
+### Displaying the result on screen
+
+We're almost there! In the file `Synthesis.js` (which generates the product page), you can add some code to display the `rate`. Knowing React.js or at least HTML syntax is necessary to perform this step without impacting other items on the page.
+
+```diff
+// Many more things here
+const ProductSynthesis = props => {
+  const {
+    product
+    // Many more things here
+  } = props;
+  return (
+    <div className="product__synthesis">
+      {/* Many more things here */}
++     <div>Rate: {product.rate} / 5</div>
+      {/* Many more things here */}
+    </div>
+  );
+};
+
+export default ProductSynthesis;
+```
```

Comment:
Could you please add a conclusion about this documentation?

A lead could be to suggest to the user to add a custom attribute to a category. This will be the exact same steps as here and will help them to make sure they've understood the concepts correctly.